notebooks/index.ipynb

```diff
@@ -6,36 +6,31 @@
    "source": [
```

Comment:
<p>I like the rewording of "basic/advanced" and the restructuring of this section. :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>+1</p>